config.toml

```diff
@@ -1,4 +1,4 @@
-testing = false
```

Comment:
I don't think we should make this change - it'll probably clobber the config that's present on the server.

Reply:
More generally, though, I think we should probably figure out a better way to do this integration testing - we could talk about that next week though.